num_of_buttons: 15 
axis_linear_x: 1
axis_linear_y: 0
axis_angular: 2
scale_linear_x: 1.0
scale_linear_y: 1.0
scale_angular: 2.0
button_speed_up: 11
button_speed_down: 12
button_dead_man: 0 
button_output_1: 7
button_output_2: 8
output_1: 0
output_2: 2
button_ptz_tilt_up: 5
button_ptz_tilt_down: 6
button_ptz_pan_right: 3
button_ptz_pan_left: 4
